modules/apps/notification/notification-test/src/testIntegration/java/com/liferay/notification/internal/type/test/EmailNotificationTypeTest.java

```diff
@@ -127,18 +200,11 @@ public void testSendNotification() throws Exception {
 				}
 			},
 			ObjectDefinitionConstants.SCOPE_COMPANY);
+	}
 
-		List<NotificationQueueEntry> notificationQueueEntries =
-			notificationQueueEntryLocalService.getNotificationEntries(
-				NotificationConstants.TYPE_EMAIL,
-				NotificationQueueEntryConstants.STATUS_SENT);
-
-		Assert.assertEquals(
-			notificationQueueEntries.toString(), 1,
-			notificationQueueEntries.size());
-
-		NotificationQueueEntry notificationQueueEntry =
-			notificationQueueEntries.get(0);
+	private void _assertNotificationRecipientSettings(
```

Comment:
This method is used once and inside another private method. I believe we can add this logic in `_assetNotificationQueueEntry`. What do you think?

Reply:
Agree.